docs/guides/code-server.md

```diff
@@ -64,15 +64,13 @@ codeserver:
   image: codercom/code-server:latest
   restart: always
   ports:
-    - 8443:8443
+    - 8080:8080
   volumes:
     - ./code-server:/home/coder/project
-  command: --allow-http --no-auth --disable-telemetry
+  command: --auth none --disable-telemetry /home/coder/project
 ```
 
-Note we are mounting a directory called`./code-server`. Be sure to give the default docker user write permissions to that folder by running `chown -R 1000:1000 code-server/`.
-
```

Comment:
When applied, this ended up making the filesystem unwritable for me. Worked well without.

I may be missing something, but I know our Dockerfile now [fixes a debian uid error](https://github.com/cdr/code-server/blob/main/ci/release-image/Dockerfile#L28-L33).